### DISCUSSION TITLE
Surface selection from Gmsh

### DISCUSSION THREAD

**Discussion by drozp94** [OpenRadioss/OpenRadioss#3194] [Q&A] [answered]
Hello my friends,

I am working on some bending simulation and I want to apply the force on the upper pusher, as I don't know yet how to user RBEs (I am using just Open Source tools), I decided to use pressure load on a the upper surface /PLOAD

/PLOAD/1
Pressure load on pusher 1
#  surf_ID   fct_IDT   sens_ID                                           AscaleX             FscaleY
      1005         1         0                                                 1                  -1

Some more details about my model: I am using 3D elements with a property of TSHELL.

the problem which I am facing is related to the Gmsh export. When I export mesh I select in the prompt window to ignore the lines to shave Physical Surfaces and Physical Volumes. The exported "rad" file contains the definition of that surface:

/SH3N/2000005
# SET_ELEMENT=PhysicalSurface55
        22       370      9255       369
        23       369      9255       368
        24       371      9255       370
        25       368      9255       367
        26       367      9255        10
        27       372      9255       371
        28        10      9255       372

and also the definition of it's selection (as I understand)

/GRSH3N/SH3N/4
F_ND1
        22        23        24        25        26        27        28

- I've tried to use directly the selection ID 4 in the PLOAD surf_ID field and it does not work, it gives me next output:
ERROR ID :    402
** ERROR IN ELEMENT PART DEFINITION
DESCRIPTION :  
   1 PART(S) REFERENCED BY ELEMENTS DO(ES) NOT EXIST. THIS MAY CAUSE OTHER ERROR MESSAGES IN THE MODEL
   /SH3N/2000005    ELEMENT(S) DEFINED IN NON EXISTING PART ID 2000005

ERROR ID :   3066
** ERROR IN PRESSURE LOAD DEFINITION (SURFACE)
DESCRIPTION :  
   -- PRESSURE LOAD ID: 1
   -- PRESSURE LOAD TITLE: Pressure load on needle 1
   NO SURFACE REFERENCED IN THE OPTION

- I've tried to use also the ID 2000005 and it also didn't worked, with the same output.
- I've tried to create a surface with /SURF/GRSH3N/1005 by selecting first the ID 4 (didn't worked), second ID 2000005 (didn't worked).
- It always says: NO SURFACE REFERENCED IN THE OPTION
- As I understand the software asks me to create /PART definition for that surface /SH3N/2000005, but here is my confusion, as this surface is a part of a 3D part, it is not a separate 2D part.

As a conclusion, I've tried several things until now and I cannot figure it out yet. I would be really thankful if somebody can help me with some hints how to use defined physical surfaces for loads or boundary  conditions from Gmsh in OpenRadioss. I know I am doing something wrong, but I don't know what yet.

P.S. I also tried the CLOAD with group of nodes and it works, but as I don't know yet how to use RBEs, the force is applied for each node and the total force can be different function of the node number. 

Really appreciate your help!
Dima

**Comment by PaulAltair** [comment, marked as answer]
Yes, the output from GMSH in this mode is exporting the 2d surface elements used in construction of the tetra mesh, but no corresponding part, for the PLOAD, you need to reference a /SURF, you may look at : [/SURF/SEG in Radioss help](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/surf_seg_starter_r.htm)

and reformat your 

```
/SH3N/2000005
        22       370      9255       369
        23       369      9255       368
```
as:

```
/SURF/SEG/2000005
My surface
        22       370      9255       369
        23       369      9255       368
```
Then the ref 2000005 should work on your PLOAD ) you don't need the /GRSH3N, /SH3N entries

Check what you get, if the normals of the segments are ok, then pressure should apply as expected, if it is going the wrong way, you may need to scale your pressure x -1

**Reply by drozp94** [reply]
Hello Paul,

Thanks a lot for your reply and help! I will try it right now!

Dima

**Comment by drozp94** [comment]
Hello again,

All works, thank you very much!

Dima
